components/Header/Header.tsx

```diff
@@ -48,21 +66,45 @@ export function Header() {
         </HomeLinkAndPageDescriptionWrapper>
         <WalletAndMenuWrapper>
           {showDelegationNotification && (
-            <DelegationNotificationWrapper style={delegationNotificationStyle}>
+            <NotificationWrapper style={delegationNotificationStyle}>
               {isDelegatePending && (
-                <DelegationNotificationIconWrapper>
-                  <DelegationNotificationIcon />
-                </DelegationNotificationIconWrapper>
+                <NotificationIconWrapper>
+                  <DelegateRequestNotificationIcon />
+                </NotificationIconWrapper>
               )}
-              <DelegationNotificationText>
-                <NextLink href="/wallet-settings" passHref>
+              <NotificationText>
+                <Link href="/wallet-settings">
                   <>
-                    {isDelegatePending && <A>Received request</A>}
-                    {isDelegate && <A>Delegate connected</A>}
+                    {isDelegatePending && "Received delegate request"}
+                    {isDelegate && "Delegate connected"}
                   </>
-                </NextLink>
-              </DelegationNotificationText>
-            </DelegationNotificationWrapper>
+                </Link>
+              </NotificationText>
+              <MobileNotificationText>
+                <Link href="/wallet-settings">Delegate</Link>
+              </MobileNotificationText>
+            </NotificationWrapper>
+          )}
+          {showV1RewardsNotification && (
+            <NotificationWrapper style={v1RewardsNotificationStyle}>
+              <NotificationIconWrapper>
+                <V1RewardsNotificationIcon />
+              </NotificationIconWrapper>
+              <NotificationText>
+                <Strong>
+                  {formatNumberForDisplay(v1Rewards?.totalRewards)}
+                </Strong>{" "}
+                UMA from v1{" "}
```

Comment:
i wonder if this should be a link to the old app, readme or something that explains it

Reply:
I think we should avoid pointing people to the old one actually...it wont work :) you cant claim rewards on the old contract once migrated and _must_ claim them via the new contract using the method done here.